.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,30 @@
+---
+name: "Bug report \U0001F41E"
```

Comment:
Wait, can you use the same emoji here as React?

Reply:
I think I did.. idk, why it doesnt show here...

Reply:
![image](https://user-images.githubusercontent.com/81580355/115241762-da1d0c80-a129-11eb-8622-b7fab7dda6e2.png)

It shows, if you go to md..

Reply:
changed